shell/platform/android/io/flutter/view/FlutterView.java

```diff
@@ -547,18 +547,29 @@ private void updateViewportMetrics() {
     private void handlePlatformMessage(String channel, String message, final int responseId) {
         OnMessageListener listener = mOnMessageListeners.get(channel);
         if (listener != null) {
-            nativeInvokePlatformMessageResponseCallback(mNativePlatformView, responseId, listener.onMessage(this, message));
+            String response;
+            try {
+                response = listener.onMessage(this, message);
+            } catch (Exception ex) {
+                Log.e(TAG, "Exception in message listener", ex);
```

Comment:
We might want to say "Uncaught exception in message listener", since this is the place where the exception will be suppressed.